nixos/modules/services/web-apps/goatcounter.nix

```diff
@@ -0,0 +1,80 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  inherit (lib) types;
+  cfg = config.services.goatcounter;
+  stateDir = "goatcounter";
+in
+
+{
+  options = {
+    services.goatcounter = {
+      enable = lib.mkEnableOption "goatcounter";
+
```

Comment:
a bit more description would be nice, like
`goatcounter, a service to easily track web analytics`

Reply:
* https://github.com/NixOS/nixpkgs/pull/327184#discussion_r1679969103

Reply:
also, it's easy to change tab to "packages" on https://search.nixos.org/